updater/lib/dependabot/sentry/sentry_context_processor.rb

```diff
@@ -0,0 +1,25 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "sorbet-runtime"
+
+require "dependabot/sentry/processor"
+
+class SentryContext < ::Dependabot::Sentry::Processor
+  sig do
+    override
+      .params(
+        event: Sentry::Event,
+        hint: T::Hash[Symbol, T.untyped]
+      )
+      .returns(Sentry::Event)
+  end
+  def process(event, hint)
+    if (exception = hint[:exception])
+      exception.raven_context.each do |key, value|
+        event.send("#{key}=", value)
+      end
+    end
```

Comment:
Copied from the migration guide here: https://docs.sentry.io/platforms/ruby/migration/#exceptionraven_context